.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Bugs
+about: Crashes and other bugs
+title: ''
+labels: 'bug'
```

Comment:
Up until now I've only labelled issues as "bugs" until the bug report was complete (debug logs, enough information to start debugging). I think it's a good idea to have a label for all bugs, but I wonder if we should have a replacement for the old scheme. Maybe I can just add the "waiting on reporter" label when typing out the message asking for missing info.

Reply:
My vote would be to use `waiting on reporter` for that. I don't think `bug` is the correct label to use for that.

Reply:
Works for me!